aviary/examples/external_subsystems/engine_NPSS/NPSS_Model/DesignEngineGroup.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import openmdao.api as om
+import subprocess as subprocess
+from openmdao.utils.file_wrap import FileParser
+from aviary.subsystems.propulsion.engine_sizing import SizeEngine
+import time
+import os
+
+from aviary.examples.external_subsystems.engine_NPSS.engine_variables import Aircraft, Dynamic
+
+# Number of points in NPSS model deck. 
+# Will need to be updated if the size of the deck changes.
+vec_size = 72
+
+class NPSSExternalCodeComp(om.ExternalCodeComp):
+    def setup(self):
+        self.add_input('Alt_DES', val=0.0, units='ft')
+        self.add_input('MN_DES', val=0.0, units=None)
+        self.add_input('W_DES', val=240.0, units='lbm/s')
```

Comment:
more descriptive names are prefered, maybe design_altitude, design_mach, and design_airflow?

Reply:
Fixed and uploaded.